### PR TITLE
Add bank account

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: src/Entities/Schema.php [hunk 1]
```diff
@@ -30,6 +30,15 @@
     public function __construct()
     {
         $this->builders = [
+            'bank-accounts' => function (array $content) {
+                return new Collection(new BankAccount(), $content);
+            },
+            'bank-accounts/{bankAccountId}' => function (array $content) {
+                return new BankAccount($content);
+            },
+            'bank-accounts/{bankAccountId}/deactivation' => function (array $content) {
```
Comment (slavcodev) [inline src/Entities/Schema.php]: I think on this endpoint we cannot get any resource, so this builder is unnecessary.

File: tests/Api/BankAccountTest.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * Class BankAccountTest.
+ *
+ * @author Dara Pich <dara.pich@rebilly.com>
+ */
+class BankAccountTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function searchTheContactTests()
```
Comment (slavcodev) [inline tests/Api/BankAccountTest.php]: I think it's copy-paste mistake here? :)

File: src/Entities/Schema.php (hunk 1 above)
Comment (dara123) [inline src/Entities/Schema.php]: thanks

File: tests/Api/BankAccountTest.php (hunk 2 above)
Comment (dara123) [inline tests/Api/BankAccountTest.php]: sorry. wrong function name

**Comment by slavcodev** [comment]
Looks ok.

**Pull request merged**